src/executor/exec_pass.h

```diff
@@ -198,6 +198,29 @@ Graph InferStorageType(Graph&& graph,
                        StorageTypeVector&& storage_type_inputs = StorageTypeVector(),
                        const std::string& storage_type_attr_key = "");
 
+/*! \brief The default storage type inference function, which assigns all undefined
+ *         storage types to kDefaultStorage. If all of input and output storage types
+ *         are kDefaultStorage, DispatchMode::kFCompute is assigned to dispatch_mode. Otherwise,
+ *         DispatchMode::kFComputeFallback is assigned to dispatch_mode.
+ */
+bool DefaultStorageType(const nnvm::NodeAttrs& attrs,
```

Comment:
This declaration seems to be duplicated from here: https://github.com/apache/incubator-mxnet/blob/0d5ebe169c61f2f12daacfdc16e575f16dc9cb21/src/common/exec_utils.h#L224  I wonder if it's been refactored and this is a merge error?